src/settings/settings.go

```diff
@@ -11,18 +11,34 @@ type Settings struct {
 	// runtime options
 	GrpcUnaryInterceptor grpc.ServerOption
 	// env config
-	Port                         int           `envconfig:"PORT" default:"8080"`
-	GrpcPort                     int           `envconfig:"GRPC_PORT" default:"8081"`
-	DebugPort                    int           `envconfig:"DEBUG_PORT" default:"6070"`
-	UseStatsd                    bool          `envconfig:"USE_STATSD" default:"true"`
-	StatsdHost                   string        `envconfig:"STATSD_HOST" default:"localhost"`
-	StatsdPort                   int           `envconfig:"STATSD_PORT" default:"8125"`
-	RuntimePath                  string        `envconfig:"RUNTIME_ROOT" default:"/srv/runtime_data/current"`
-	RuntimeSubdirectory          string        `envconfig:"RUNTIME_SUBDIRECTORY"`
-	RuntimeIgnoreDotFiles        bool          `envconfig:"RUNTIME_IGNOREDOTFILES" default:"false"`
-	RuntimeWatchRoot             bool          `envconfig:"RUNTIME_WATCH_ROOT" default:"true"`
-	LogLevel                     string        `envconfig:"LOG_LEVEL" default:"WARN"`
-	LogFormat                    string        `envconfig:"LOG_FORMAT" default:"text"`
+	Port      int `envconfig:"PORT" default:"8080"`
```

Comment:
The reason for splitting settings.go into different groups is to reduce mere conflicts

Previously there were two reasons for merge conflicts:
- gofmt could reformat all the settings when a new setting is add
- people tend to add new settings to the end, which has a high likelihood of merge conflicts if multiple people are adding new settings